package.json

```diff
@@ -28,23 +28,25 @@
   ],
   "dependencies": {
     "debug": "^4.3.4",
-    "jose": "^4.14.4",
     "joi": "^17.9.2",
+    "jose": "^4.14.4",
     "openid-client": "^5.4.3"
   },
   "devDependencies": {
     "@types/debug": "^4.1.8",
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^29.5.3",
     "@types/node": "^20.4.4",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "@typescript-eslint/parser": "^5.62.0",
-    "eslint": "^8.45.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^27.5.1",
-    "prettier": "^2.8.8",
-    "ts-jest": "^27.1.4",
+    "@typescript-eslint/eslint-plugin": "^6.1.0",
+    "@typescript-eslint/parser": "^6.1.0",
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^8.10.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "jest": "^29.6.2",
+    "jest-mock": "^29.6.2",
+    "prettier": "^3.0.1",
+    "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5"
-  }
+    "typescript": "^5.1.6"
```

Comment:
Double check some of these deps, as these could also use with some version updating, since making this change.